src/renderer/screens/config/ConfigScreen.scss

```diff
@@ -8,6 +8,7 @@
   left: 0px;
   right: 0px;
   width: 100%;
+  overflow: auto;
```

Comment:
These were some drive-by fixes to CSS layout issues. Mainly due to the new error message that can cause the screen to overflow the y axis.